render/wlr_texture.c

```diff
@@ -22,6 +22,8 @@ void wlr_texture_destroy(struct wlr_texture *texture) {
 struct wlr_texture *wlr_texture_from_pixels(struct wlr_renderer *renderer,
 		uint32_t fmt, uint32_t stride, uint32_t width, uint32_t height,
 		const void *data) {
+	assert(width > 0);
+	assert(height > 0);
```

Comment:
Maybe also assert that `stride > 0`?

Reply:
`assert(data)` as well?

Reply:
And `stride >= width` rather than `stride > 0`?

Reply:
Yeah, I guess that makes sense too

Reply:
I'm fine with `stride >= width` yeah, even if it won't catch all errors.